modules/nw-allocate-load-balancers-to-subnets-procedure.adoc

```diff
@@ -0,0 +1,121 @@
+
+** Subnets can fulfill multiple roles. For example, a subnet could host the IngressControllerLB and also be used for Bootstrap. You will define these roles in the `install-config.yaml` file.
+
+** It is  highly recommended that you use subnets spanning at least two availability zones (AZs) for high availability of load balancers and other critical components like control planes.
+
+** Sufficient available IP addresses within these subnets for all assigned roles.
+
+** The AWS configuration for these subnets, including Network ACLs and Security Groups, must permit necessary traffic for all roles assigned to them. For subnets hosting an Ingress Controller, this typically includes TCP ports 80 and 443 from desired sources.
+
+* The OpenShift installer binary for your target OpenShift version.
+
+* An `install-config.yaml` file.
+
+.Procedure
+
+. Prepare the `install-config.yaml` file:
++
+If you haven't already, generate the installation configuration file using the OpenShift installer:
++
+[source,terminal]
+----
+$ openshift-install create install-config --dir=<your_installation_directory>
+----
++
+This command creates the `install-config.yaml` file in the specified directory.
+
+. Define Subnets and assign roles:
++
+Open the `install-config.yaml` file located in `<your_installation_directory>` using a text editor. You will define your VPC subnets and their designated `roles under platform.aws.vpc.subnets` field.
++
+For each AWS subnet you intend the cluster to use, you will create an entry specifying its ID and a list of roles. Each role is an object with a type key. To designate a subnet for the default Ingress Controller, assign it a role with type: `IngressControllerLB`.
++
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: example.com <1>
+metadata:
+name: my-cluster # Example cluster name
+platform:
+    aws:
+        region: us-east-1 <2>
+        vpc: <3>
+        subnets: <4>
+        - id: subnet-0fcf8e0392f0910d5 <5>
+            roles:
+            - type: IngressControllerLB <6>
+            - type: Bootstrap
+        - id: subnet-0xxxxxxxxxxxxxxza
+            roles:
+            - type: IngressControllerLB 
+        - id: subnet-0fcf8e0392f0910d4 # Private Subnet in AZ us-east-1a
+            roles:
+            - type: ClusterNode <7>
+        - id: subnet-0yyyyyyyyyyyyyyzb 
+            roles:
+            - type: ClusterNode
+pullSecret: '...' <8>
+sshKey: '...' <9>
+----
+<1> Your base domain.
+<2> Your AWS region.
+<3> The vpc object under platform.aws contains the subnets list.
+<4> List of all subnet objects that OpenShift will utilize. Each object defines a subnet id and its roles.
+<5> Replace with your actual AWS Subnet ID. 
+<6> The type: IngressControllerLB role specifically designates this subnet for the default Ingress Controller's LoadBalancer. This subnet must be public.
```

Comment:
In private/internal cluster, the subnet with `IngressControllerLB` must be private.